girder_worker_tasks/app_support/__init__.py

```diff
@@ -0,0 +1,28 @@
+from girder_worker import GirderWorkerPluginABC
+from girder_worker.app import app
+from girder_worker.utils import girder_job
+from .pgls import pgls
```

Comment:
from .app_support import *
from .app_suppor2 import *

Reply:
to support import *, within app_support/__init__.py we need

__all__ = (
pgsr,
asr
)